src/Protocols/Wishbone/Standard/Hedgehog.hs

```diff
@@ -0,0 +1,317 @@
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE RecordWildCards #-}
+{-# OPTIONS_GHC -Wno-name-shadowing #-}
```

Comment:
We check for shadowing to prevent "obvious" mistakes. If you need multiple variables with the "same" name, suffix them with `[0..n]`.